lib/conduit_lwt_unix.ml

```diff
@@ -175,56 +175,33 @@ module Sockaddr_client = struct
         Lwt_unix.connect fd sa >>= fun () ->
         let ic = Lwt_io.of_fd ~mode:Lwt_io.input fd in
         let oc = Lwt_io.of_fd ~mode:Lwt_io.output fd in
-        return (fd, ic, oc)
+        Lwt.return (fd, ic, oc)
       )
 end
 
 module Sockaddr_server = struct
-  open Conduit_lwt_server
-
-  let init_socket ?(backlog=128) sockaddr =
-    Unix.handle_unix_error (fun () ->
```

Comment:
@avsm I think you added this line originally. I had to get rid of it since the other Lwt backends didn't use it. I don't think exiting the program is good behavior anyway.